.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy
+
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set environment variables
+        run: |
+          echo "DEPLOY_BUILD_NUMBER=$((500 + ${{ github.run_number }}))" >> $GITHUB_ENV
+      - name: Deploy branch snapshot
+        if: "!startsWith(github.ref, 'refs/tags/')"
```

Comment:
forking this repo may lead to failures when someone tries to build it, maybe we shouldn't run it when it's a fork?

Reply:
Added `github.repository_owner == 'meniga'` condition for job